packages/block-library/src/cover/style.scss

```diff
@@ -104,7 +104,13 @@
 	.wp-block-cover__inner-container {
 		width: 100%;
 		z-index: z-index(".wp-block-cover__inner-container");
-		color: $white;
+
+	}
+
+	&.is-dark-theme {
```

Comment:
The class is-dark-theme is only applied on the editor side, not on the frontend (not found by searching in packages/block-library/src/cover/save.js). Styles that apply only on the editor side should be on "packages/block-library/src/cover/editor.scss", to avoid loading styles on the frontend that are not used.

Reply:
This has been problematic in the past. It's easy to confuse the class, discovered via inspecting in the editor, and assume it's also on the frontend, when as you suggest it's only about the block UI being contrasty (such as a white focus style for blocks when the theme is dark, vs. a blue focus style). The class has also been confused with being related to operating system dark modes, whether for the theme or for the editor. 

We might need to reconsider that UI inversion feature entirely, either automate it so it works without a CSS class, or find some other way to explore contrast in such cases.